src/components/review/ReviewsList.js

```diff
@@ -67,10 +68,20 @@ const AlbumReviews = ({ albumId }) => {
       {user && userHasReviewed && (
         <p> You already submitted the review for this album</p>
       )}
+
       {/* Conditionally render the WriteReview component */}
       {user && !userHasReviewed && (
-        <WriteReview albumId={albumId} refreshReviews={refreshReviews} />
+        <WriteReview
+          albumId={albumId}
+          refreshReviews={refreshReviews}
+          userHasReviewed={userHasReviewed}
+          reviewId={reviews[0]?._id}
```

Comment:
Currently, the `WriteReview` component doesn't seem to accept these props (`userHasReviewed` and `reviewId`). Can they be removed?